app/helpers/template_helpers.py

```diff
@@ -112,10 +112,11 @@ def page_header_context(self) -> dict[str, Union[bool, str, LazyString]]:
         context: dict[str, Union[bool, str, LazyString]] = {
             "orgLogo": f"{self._survey_config.page_header_logo}",
             "orgLogoAlt": f"{self._survey_config.page_header_logo_alt}",
+            "title": self._survey_title
+            if self._survey_title and cookie_session.get("theme")
+            else "ONS Surveys",
```

Comment:
The title is being set in two different ways now. You have this "title" inside the header context but we also set the title in the page config in the `_base.html`. Need to find out which way it should be done. Currently they are both being set. It currently works because at least one or the other is set, but we need to understand when they should be used and whether both should be used.

Reply:
They do overlap because you introduced the `title` in the header context in this PR but the title for the banner already works.

Reply:
But I didn't introduce `title` in header context as it was already:
```python
if self._survey_title:
            context["title"] = self._survey_title
```
Do you mean I introduced default for the title in header context?

Reply:
From what I can see if we don't define it template helpers it is taken from env var `SURVEY_TYPE` and if that is not defined it defaults in settings.py to `BusinessSurveyConfig` survey_title since `SURVEY_TYPE = os.getenv("SURVEY_TYPE", "business")` in `setttings.py`

Reply:
If we don't have it defined in context (as we did before) it uses value from the template which is different as it uses both page title and survey title instead of just survey title in the header element.

Reply:
> But I didn't introduce `title` in header context as it was already:
> 
> ```python
> if self._survey_title:
>             context["title"] = self._survey_title
> ```
> 
> Do you mean I introduced default for the title in header context (and the default was taken from somewhere else before)?

Ah okay. So the duplicate title setting is an existing issue, therefore, happy to leave it here.